src/systems/collada_world_exporter/ColladaWorldExporter.cc

```diff
@@ -244,9 +246,48 @@ class ignition::gazebo::systems::ColladaWorldExporterPrivate
       return true;
     });
 
+    std::vector<common::ColladaLight> lights;
+    _ecm.Each<components::Light,
+              components::Name>(
+    [&](const Entity &/*_entity*/,
+        const components::Light *_light,
+        const components::Name *_name)->bool
+    {
+      std::string name = _name->Data();
+      const auto& sdf_light = _light->Data();
+
+      common::ColladaLight p;
+      p.name = name;
+      if (sdf_light.Type() == sdf::LightType::POINT) {
+        p.type = "point";
+      } else if (sdf_light.Type() == sdf::LightType::SPOT) {
+        p.type = "spot";
+      } else if (sdf_light.Type() == sdf::LightType::DIRECTIONAL) {
+        p.type = "directional";
+      } else {
+        p.type = "invalid";
+      }
+
+      p.position = sdf_light.RawPose().Pos();
```

Comment:
I just noticed that the light struct added to `ign-common` doesn't have the orientation. The convention we follow is that the direction is applied on top of the pose, so we need to use `RawPose().Rot()` somewhere.

See more context in https://github.com/ignitionrobotics/ign-gazebo/issues/329

Reply:
I think for now we'll have to calculate a new direction using the pose's rotation

Reply:
apply orientation to local light direction via RotateVec 415d6ac